go.mod

```diff
@@ -17,6 +17,7 @@ require (
 	github.com/openshift/api v3.9.1-0.20191111211345-a27ff30ebf09+incompatible
 	github.com/openshift/cluster-node-tuning-operator v0.0.0-20200914165052-a39511828cf0
 	github.com/openshift/custom-resource-status v0.0.0-20200602122900-c002fd1547ca
+	github.com/openshift/library-go v0.0.0-20211220195323-eca2c467c492
```

Comment:
why another lib bump? can we squash in the previous?

Reply:
Absolutely let me squash things up, the bump is to just make sure we're on the latest version that pulls in this [PR 1273](https://github.com/openshift/library-go/pull/1273)